Cargo.toml

```diff
@@ -10,25 +10,44 @@ categories = ["web-programming::http-client", "network-programming"]
 keywords = ["http", "client", "request"]
 edition = "2018"
 
+#[dependencies.native-tls]
+#version = "^0.2"
+#optional = true
+
 [dependencies]
-unicase = "^2.6"
+# sgx dependencies
+#sgx_tstd = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true ,features = ["net", "untrusted_time"]}
+sgx_tstd = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master",optional = true ,features = ["net", "untrusted_time"]}
 
-[features]
-default = ["native-tls"]
-rust-tls = ["rustls", "webpki", "webpki-roots"]
+# sgx enabled external libraries
+unicase_sgx      = { package = "unicase",        git = "https://github.com/mesalock-linux/unicase-sgx", optional = true}
+rustls_sgx       = { package = "rustls",         git = "https://github.com/mesalock-linux/rustls",                   branch = "mesalock_sgx",        optional = true }
+webpki_sgx       = { package = "webpki",         git = "https://github.com/mesalock-linux/webpki",                   branch = "mesalock_sgx",        optional = true }
+webpki-roots_sgx = { package = "webpki-roots",   git = "https://github.com/mesalock-linux/webpki-roots",             branch = "mesalock_sgx",        optional = true }
```

Comment:
We don't tab-align the dependencies (anymore), it's too much effort and questionable benefit.